http-server/src/main/java/org/triplea/web/socket/InetExtractor.java

```diff
@@ -15,11 +15,16 @@
 
   @SuppressWarnings("UnstableApiUsage")
   public static InetAddress extract(final Map<String, Object> userSession) {
-    // expected format '/127.0.0.1:42840'
-    final String ipString = String.valueOf(userSession.get(IP_ADDRESS_KEY)).substring(1);
+    // expected format '/127.0.0.1:42840' or (for test-cases) '127.0.0.1'
```

Comment:
I don't think it's a good thing to get different values from test cases that we make special cases for in non-test code.

Reply:
Generally agree. I'm on the fence as it's a huge PITA for test code to inject values in the right format and creates a lot of coupling to a specific format. Probably better to create a test utility method for less coupling; So for sure this is hack, but the cost of the hack vs ROI of better test utility method is not clear.